composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "easyengine/secure-command",
+    "description": "",
```

Comment:
Please add a description. Something like - _Configure HTTP auth for EasyEngine site._